tools/pot/openvino/tools/pot/graph/utils.py

```diff
@@ -219,3 +229,30 @@ def add_new_ignored_params(model, node, quantize_agnostic, ignored_params, model
 
 def is_data_type_quantizable(type_node):
     return type_node not in (np.int32, np.int64, bool)
+
+
+def get_hardware_config_operation_type(node, available_types):
```

Comment:
Is there any reason for moving this function from `algorithms/quantization` to `graph`?

Reply:
This is a change from the previous PR. So I don't know the exact reason, but here is my guess.

Previously,
the get_hardware_config_operation_type functions is implemented in 'openvino/tools/pot/algorithms/quantization/utils.py'.
this function is only used on 'openvino/tools/pot/algorithms/quantization/fake_quantize_configuration.py'.

By this PR,
This function is also used in 'openvino/tools/pot/graph/passes.py'.

AFAIK, 'graph' is a higher module than 'quantization'. So 'graph' should be working independently to 'quantization'.
I guess that this is the reason why the implementation of the 'get_hardware_config_operation_type' function is moved to 'openvino/tools/pot/graph/utils.py'